packages/fxa-payments-server/src/routes/Subscriptions/Cancel/en.ftl

```diff
@@ -11,3 +11,6 @@ sub-item-cancel-msg =
 sub-item-cancel-confirm =
     Cancel my access and my saved information within
     { $name } on { $period }
+
+## $promotion_name (String) - The name of the promotion.
+sub-promo-coupon-applied = {$promotion_name} coupon applied:&nbsp;
```

Comment:
```suggestion
sub-promo-coupon-applied = { $promotion_name } coupon applied:
```

Reply:
Actually, as I look at the mockups - this string is being concatenated with another. It's [best practice](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#avoid-concatenations-use-placeholders-instead) to avoid this practice and use a placeholder in the string instead.

Reply:
Intentional, but I'll remove the space from the id and revise styling. Thanks!